omnigan/data.py

```diff
@@ -242,8 +242,11 @@ def tensor_loader(path, task, domain):
         if Path(path).suffix == ".npy":
             arr = np.load(path)
         else:
-            arr = imread(path)  # .astype(np.uint8)
-        arr = torch.from_numpy(arr)
+            if domain == "r":
+                arr = np.array(Image.open(path).convert("L"))
```

Comment:
Did you check if the array made sense? When using MiDaS for smog, I tried convert("L") but it gave me images all white except a few black pixels only. What worked for me was:
`depth = Image.open(depth_path).convert("I")
   depth = np.array(depth)
    depth = normalize(depth)`

Reply:
I actually did not need to convert as I just normlize the input